src/main/java/de/tum/in/www1/artemis/domain/ProgrammingExercise.java

```diff
@@ -69,6 +69,8 @@
 @JsonInclude(JsonInclude.Include.NON_EMPTY)
 public class ProgrammingExercise extends Exercise {
 
+    // TODO: delete publish_build_plan_url from exercise using liquibase
+
```

Comment:
Liquibase Changelog needed

Reply:
Feedback Stephan: Planned after the Migration cleanup to prevent additional releases.